### DISCUSSION TITLE
HEIMAN HS2WD-E "Clone" Tuya TS0216 _TYZB01_8scntis1

### DISCUSSION THREAD

**Discussion by okastl** [Koenkk/zigbee2mqtt#8039] [Devices]
Hi there,
I have bought this "clone" of the supported HEIMAN HS2WD-E siren:
https://www.aliexpress.com/item/1005002482465860.html

It identifies itself as modelID: 'TS0216', manufacturerName: '_TYZB01_8scntis1'

I have taken the original Heiman definition, added a fingerprint and created an external converter, see attached file.
It works fine.

Maybe someone more skilled can add this to the zigbee herdsman converters. I don't dare, because I don't know, which side effects to expect as 'TS0216' exists as a Tuya siren and because of the original HEIMAN HS2WD-E, which I don't want to break for others.

[_TYZB01_8scntis1.zip](https://github.com/Koenkk/zigbee2mqtt/files/6816147/_TYZB01_8scntis1.zip)

**Comment by Koenkk** [comment]
Added, thanks!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

**Comment by ipm79** [comment]
Hi,

I just seem to have found another clone of the same device with the same issue (also identifies as Tuya). I could get it to work by using @okastl external converter file and changing the manufacturerName to "_TYZB01_4obovpbi". Amazon link to device: https://www.amazon.de/gp/product/B0816W53K1/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1

**Reply by snipah** [reply]
Hi! Bought the same siren. Basically it is identified correctly, but I can't control the sirene correctly via the exposes page or via MQTT. Mode triggers the audible alarm,but the strobe does not work for me. Also there a no switches exposed in Home Assistant, only the battery-sensor. What works for you and what does not?

@Koenkk Any idea why things don't work correctly? Also could you add the vendor the branches?

**Reply by Koenkk** [reply]
`_TYZB01_8scntis1` will be detected as HS2WD-E now. Alarms are indeed not discovered to Home Assistant, you have to trigger it by publishing a MQTT message (this can be done by a HA service call). 

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by snipah** [reply]
We would also need the vendor _TYZB01_4obovpbi! Big Thanks

**Reply by Koenkk** [reply]
done
